Apps/frontend/src/components/ControlColumnHeader.svelte

```diff
@@ -0,0 +1,24 @@
+<script>
+  export let label;
+  export let icon;
+  export let tooltip = null;
+  import { Tooltip } from "sveltestrap";
+  let div;
+</script>
+
+<div bind:this={div} style="cursor: help;">
+  <div class="icon icon--{icon}" />
+  <div>{label}</div>
+</div>
+<Tooltip target={div} placement="bottom">
+  {tooltip != null ? tooltip : label}
+</Tooltip>
+
+<style>
```

Comment:
Can you please move all styling into the app.scss so we have one single file for styling.
We already have an icon class, think we can simply move it in there.

Reply:
Thank you for the feedback! Great idea. I will do that :)